docs/reference/gettingstartedexample/gse-tap-help.md

```diff
@@ -0,0 +1,239 @@
+- **Log I2C to Disk** - Enable to log I2C timestamps to a TDMS file.
+- **Log GPIO to Disk** - Enable to log GPIO timestamps to a TDMS file.
+
+### Serial Channel Tab
+- **CSI-2 Data Source** - Selects the CSI-2 data source type. "Corrected" returns the received data after correcting for single-bit transmission errors. "Raw" returns the received data as is. 
+- **Deserializer (Input) Configuration Script** - Full path to a script file used to configure the deserializer.
+  #### Table of PXIe-148X Deserializer Tap Scripts
+
+  | **Interface Module**                                 | **Configuration Script**                              |
+  |------------------------------------------------------|-------------------------------------------------------|
+  | PXIe-1486 (4 In 4 Out - 953/954 SerDes)              | Host\\Scripts\\DS90UB954\\Tap\\Des_Tap.py             |
+  | PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes)          | Host\\Scripts\\MAX9296A\\Tap\\RAW12_ID01_Des_Tap.cpp  |
+  | PXIe-1487 (4 In 4 Out - 96717/96716A SerDes - KU11P) | Host\\Scripts\\MAX96716A\\Tap\\RAW12_ID12_Des_Tap.cpp |
+  | PXIe-1488 (4 In 4 Out - 971/9702 SerDes - KU11P)     | Host\\Scripts\\DS90UB9702\\Tap\\Des_Tap.py            |
+
+- **Serializer (Output) Configuration Script** - Full path to a script file used to configure the serializer.
+  #### Table of PXIe-148X Serializer Tap Scripts
+
+  | **Interface Module**                                              | **Configuration Script**                                       |
+  |-------------------------------------------------------------------|----------------------------------------------------------------|
+  | PXIe-1486 (4 In 4 Out - 953/954 SerDes)                           | Host\\Scripts\\DS90UB953\\Tap\\Ser_Tap.py                      |
+  | PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes)                       | Host\\Scripts\\MAX9295A\\Tap\\RAW12_ID1_Ser_Tap.cpp            |
+  | PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes) Reverse Splitter Mode | Host\\Scripts\\MAX9295A\\Tap\\RAW12_ID0_ToRevSplit_Ser_Tap.cpp |
+  | PXIe-1487 (4 In 4 Out - 96717/96716A SerDes - KU11P)              | Host\\Scripts\\MAX96717\\Tap\\RAW12_ID2_Ser_Tap.cpp            |
+  | PXIe-1488 (4 In 4 Out - 971/9702 SerDes - KU11P)                  | Host\\Scripts\\DS90UB971\\Tap\\Ser_Tap.py                      |
+
+- **Channel Configurations** - Array of active serial channel tap pairs and display parameters for each channel pair.
+    - **Serial Channel Tap Pair**
+        - **input** - String representing the active serial input channel in the tap pair in the format "SI" for Serial Input followed by the channel number (i.e. "SI0")
+        - **output** - String representing the active serial output channel in the tap pair in the format "SO" for Serial Output followed by the channel number (i.e. "SO0")
```

Comment:
Optional: could add note indicating numerical value of pairs must match

Reply:
I added the note like you suggested.